frontend/src/dummyData/Contacts/rows.js

```diff
@@ -0,0 +1,42 @@
+export const rows = [
```

Comment:
these files are being used in tests and as temporary input as if it came from the api (i think) afaict. it would be nice if where they are being used was documented, either here or in a readme, so that [our future selves/new devs] understand the full scope/intent of these files.